src/Target.h

```diff
@@ -140,6 +140,11 @@ struct Target {
         return has_feature(CUDA) || has_feature(OpenCL);
     }
 
+    /** Some device apis use coordinate-based stores/loads. */
+    bool has_coordinates_feature() const {
+        return has_feature(OpenGL);
```

Comment:
I'm not sure this makes sense as a target flag. There's already logic elsewhere (inside_kernel_loop in CoordinateIntrinsics.cpp) to determine this independently...